prow/plugins/override/override_test.go

```diff
@@ -949,7 +956,7 @@ func TestWhoCanUse(t *testing.T) {
 
 	who := whoCanUse(override, "org1", "repo1")
 	if who != expectedWho {
-		t.Errorf("expected %s, got %s", expectedWho, who)
```

Comment:
This more correctly follows golang style.

I'd consider %q if you're concerned about empty strings.

Reply:
I just added a colon to make it clearer. While the messages might get long
```
    override_test.go:959: expected: Repo administrators, and the following github teams:org1/repo1: team-foo team-bar, org1: team-foo-bar., got: Repo administrators, and the following github teams:org1/repo1: team-foo team-bar, org1: team-foo-bar.
```

But using `%q` works as well that adds quotes :) 
```
    override_test.go:959: expected "Repo administrators, and the following github teams:org1/repo1: team-foo team-bar, org1: team-foo-bar.", got "Repo administrators, and the following github teams:org1/repo1: team-foo team-bar, org1: team-foo-bar."
```